borgbackup-job

```diff
@@ -101,26 +93,34 @@ init() {
   dry_run=false
   while true; do
     case $1 in
-      '--exclude')
+      --envfile|-e)
+        [[ ${2:0:1} != '-' ]] || abort "$1 requires a filename argument"
+        envfile="$2"
+        shift 2
+        ;;
+      --exclude)
         [[ ${2:0:1} != '-' ]] || abort "$1 requires an argument"
         excludes_on_cmdline+=("$2")
         shift 2
         ;;
-      '--dry-run')
+      --dry-run)
         dry_run=true
         shift
         ;;
-      '--help')
+      --help)
         show_help
         die 0
         ;;
-      '--') # Separator between optional and positional parameters
+      --) # Separator between optional and positional parameters
```

Comment:
This is no longer true. These arguments are no longer optional. We mention this on one other place in this file as well, so we should change both these since we now have required named parameters.

Reply:
These '--' is the output by getopt to signal end of getopt parsing. What follows is parameters given by the user that 
getopt does not know about so we 'break' the loop.
Changed the comment.

Reply:
not sure which other place you mean

Reply:
Yes, I know what the double dashes are for. What I think we should change is that we call these "optional" parameters. They are no longer optional. This should be changed as well:
```
# Sets double dashes, which marks end of optional parameters
eval set -- "$options"
unset options
```

I can fix it and push...